src/components/Footer.jsx

```diff
@@ -98,29 +88,23 @@ const Footer = () => {
     <StyledFooter>
       <div className="footer-content">
         <div className="footer-links">
-          <div className="donate-button-container">
-            <a
-              href="https://www.codeforamerica.org/donate-to-a-brigade?utm_campaign=Code%20for%20Chicago&utm_source=Brigade%20site"
-              target="blank"
-              rel="noopener noreferrer"
-              className="button donateButton label-3"
-            >
-              Donate
-            </a>
-
-            <img src={longArrowAltRight} alt="" />
-          </div>
+          <Button
+            text="Donate"
+            link="https://www.codeforamerica.org/donate-to-a-brigade?utm_campaign=Code%20for%20Chicago&utm_source=Brigade%20site"
+            center
+            arrow
+          />
           <ul className="internal-links p2-body">
```

Comment:
These links need the text decoration removed.

Reply:
good catch